src/lib/ngsi/ContextElementResponse.cpp

```diff
@@ -134,6 +134,7 @@ ContextElementResponse::ContextElementResponse
 (
   const mongo::BSONObj&  entityDoc,
   const AttributeList&   attrL,
+  AttributeList*         metadataList,
```

Comment:
Yeah, I tried to do that.
The compiler didn't agree with me so I made it a simple pointer